datafusion/src/sql/planner.rs

```diff
@@ -2053,8 +2058,8 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             select_list, where_clause
         );
 
-        let rewrite = DFParser::parse_sql(&query)?;
-        self.statement_to_plan(&rewrite[0])
+        let mut rewrite = DFParser::parse_sql(&query)?;
+        self.statement_to_plan(rewrite.remove(0))
```

Comment:
I realize this was not added in this PR, but I wonder if we could add an `assert_eq!(rewrite.len(), 1)` here to make sure we don't accidentally lose one of these statements later on